docs/api/classes/OSFTAlgorithm.md

```
@@ -144,6 +144,26 @@ Executes the OSFT training process.
 | `rdzv_id` | `int` | Random | Unique job ID for rendezvous. All nodes must use the same ID. |
 | `rdzv_endpoint` | `str` | Required for multi-node | Endpoint of the master node in the format `"hostname:port"`. Master node uses `"0.0.0.0:port"`, workers use master's IP address. |
 
+###### Logging Configuration
+
+Loggers are automatically enabled when their configuration parameters are set:
+
+| Logger | Enabled By | Env Variable Fallback |
+|--------|------------|----------------------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)